test/integration/billing-iam/controls/billing-iam.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Billing Bindings
+
+billing_iam_test_accounts = attribute('billing_iam_test_accounts')
+members = attribute('members')
+
+control "GCP Billing IAM" do
+            title "GCP Billing Bindings"
+            billing_iam_test_accounts.each do |billing_iam_test_accounts|
+                describe command ("gcloud beta billing accounts get-iam-policy #{billing_iam_test_accounts} --format=json") do
+                its(:exit_status) { should eq 0 }
+                its(:stderr) { should eq '' }
+
+                let!(:data) do
+                    if subject.exit_status == 0
+                        JSON.parse(subject.stdout)
+                    else
+                        {}
+                    end
+                end
+
+                describe "members" do
+                it "are bound" do
+                    transformed_data={}
+                    data['bindings'].each do |binding|
```

Comment:
This still isn't doing what I want.

I really want to emphasize that tests should really not be doing this much logic.

Instead, in tests you often want to *hard-code* the expected value. So in this case we'd want to hard code that we expect a certain SA to have a role (ex. Billing Viewer).

if you merely take the outputs for a function and then test that those outputs match the inputs you're not really testing anything.